modules/govuk/manifests/apps/content_store.pp

```diff
@@ -57,6 +57,9 @@
 # [*router_api_bearer_token*]
 #   The bearer token that will be used to authenticate with the router api
 #
+# [*draft_router_api_bearer_token*]
+#   The bearer token that will be used to authenticate with the draft router api
+#
```

Comment:
You don't need to add in this draft_router_api_bearer_token, we just need to override the hieradata of the router_api_bearer_token to use the draft one like we've done with oauth_id and oauth_secret: https://github.com/alphagov/govuk-puppet/blob/81cb1eadd1c3457df22c030b8fb2b01cdc186437/hieradata/class/draft_content_store.yaml#L11 (and hieradata_aws)

Reply:
Thank you for spotting that! I've pushed an amended commit.